File: Luyencodenet/Luyencode.net/DK08.cpp
#include <iostream>
#include <iomanip>
using namespace std;
int main() {
	float a, b;
	char c;
	cin >> a >> c >> b;
	switch (c) {
	case '+':
		cout << fixed << setprecision(2) << a + b << endl;
		break;
	case '-':
		cout << fixed << setprecision(2) << a - b << endl;
		break;
	case '*':
		cout << fixed << setprecision(2) << a * b << endl;
		break;
	case '/':
		if (b == 0) {
			cout << "Math Error";
			break;
		}
		cout << fixed << setprecision(2) << a / b << endl;
		break;
	}
	return 0;
}

File: Luyencode.net/Python/DK08.py
calc = [i for i in input().split()]
if calc[1] == '+': print(f"{float(calc[0])+float(calc[2]):.2f}")
if calc[1] == '-': print(f"{float(calc[0])-float(calc[2]):.2f}")
if calc[1] == '*': print(f"{float(calc[0])*float(calc[2]):.2f}")
if calc[1] == '/':
    if float(calc[2]) == 0: print("Math Error")
    else: print(f"{float(calc[0])/float(calc[2]):.2f}")